drift/views/v0.py

```diff
@@ -1,35 +1,30 @@
-from flask import Blueprint, current_app, jsonify, request
-import os
-import requests
-from requests import RequestException
+from flask import Blueprint, jsonify, request
 
-from drift.views.phony_data import PHONY_DATA
+from drift import info_parser
+from drift.exceptions import BadRequest
+from drift.inventory_service_interface import fetch_hosts, get_key_from_headers
 
-# TODO: manage these settings in a config.py
 APP_URL_PREFIX = "/r/insights/platform/drift"
 API_VERSION_PREFIX = "/v0"
-INVENTORY_SVC_URL = os.getenv('INVENTORY_SVC_URL', "http://inventory_svc_url_is_not_set")
-INVENTORY_SVC_PATH = '/r/insights/platform/inventory/api/v1'
 
 section = Blueprint('v0', __name__, url_prefix=APP_URL_PREFIX + API_VERSION_PREFIX)
 
 
 @section.route("/compare")
 def compare():
-    if 'X-RH-IDENTITY' in request.headers:
-        current_app.logger.debug("X-RH-IDENTITY received")
-    return jsonify(PHONY_DATA)
+    host_ids = request.args.getlist('host_ids[]')
+    auth_key = get_key_from_headers(request.headers)
+
+    if host_ids is None or host_ids == []:
```

Comment:
If I recall my python, this can be simplified to `if not host_ids`

Reply:
i just tried, you're right